SU-SOE/soe_basic
# D8CORE-2468: default action link to red

# READY FOR REVIEW

# Summary
- adding a default read to the --action link.

# Needed By (Date)
- Wednesday

# Urgency
- Med

# Steps to Test

1. Pull in this branch.
2. Build
3. add a card and select the link type to be action. 
4. add another card and select button.
5. Notice the button is turquoise and the link is red.

# Affected Projects or Products
- SOE Subtheme

# Associated Issues and/or People
- D8CORE-2468

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
